tests/rhtap-demo/rhtap-demo.go

```diff
@@ -152,7 +148,6 @@ var _ = framework.RhtapDemoSuiteDescribe(func() {
 							Expect(fw.AsKubeAdmin.HasController.DeleteAllApplicationsInASpecificNamespace(namespace, 30*time.Second)).To(Succeed())
 							Expect(fw.AsKubeAdmin.CommonController.DeleteAllSnapshotEnvBindingsInASpecificNamespace(namespace, 30*time.Second)).To(Succeed())
```

Comment:
Is this planned to be removed in another PR?

Reply:
nice catch, 
should be removed in this PR, 
removed it